swagger.yml

```diff
@@ -1022,35 +1043,48 @@ components:
         Microsoft/SQLServer:
           type: boolean
           nullable: true
+        example-1: {}
     TechnologyStatus:
       type: object
-      required:
-        - Product
-        - HostsCount
-        - Compliance
-        - UnpaidDues
       properties:
         Product:
           type: string
+          description: technologial product code
         HostsCount:
           type: integer
+          description: number of hosts found supporting the current product / total hosts found
         Compliance:
           type: number
+          description: "a metric ranging [0..1] calculated as follows:\nif a ULA exists for a given product, its compliance := 1. \n\nWhen calculating the total, this product's purchasedLicenses := usedLicenses\nelse if usedLicenses <= purchasedLicenses the product's compliance := 1. \n\nWhen calculating the total, this product's purchasedLicenses := usedLicenses \nelse this product's compliance := purchasedLicenses / usedLicenses."
         UnpaidDues:
           type: number
-    AllTechnologyComplianceStatus:
-      type: object
+          description: |-
+            "unpaidDues" = difference between calculated cost of used and purchased licenses:
+
+            if usedLicensesCost <= purchasedLicensesCost, then unpaidDues := 0
+            else unpaidDues := usedLicensesCost - purchasedLicensesCost
       required:
+        - Product
         - HostsCount
         - Compliance
         - UnpaidDues
+        example-1: {}
+    AllTechnologyComplianceStatus:
+      type: object
       properties:
         HostsCount:
           type: integer
+          description: sum of hostsCount
```

Comment:
No, it's the sum of hostsCount but the number of hosts

Reply:
ok